From cc65839b65e9a41ca0e9e633ac32a085cec66fa2 Mon Sep 17 00:00:00 2001
From: Simon McVittie <smcv@debian.org>
Date: Fri, 29 Oct 2021 10:58:13 +0100
Subject: [PATCH] virtual-package-names-list: Add dbus-system-bus,
 default-dbus-system-bus

This is the same as dbus-session-bus, but for the well-known system bus
described in the D-Bus Specification[1].

dbus, the reference implementation of D-Bus, Provides these since
bullseye. dbus-broker, an independent reimplementation, also Provides
dbus-system-bus in unstable.

The mention of "including service activation" is intended to make it
clearer that some implementation of /usr/share/dbus-1/system-services/
activation is required, for example via dbus' setuid helper and
conditional handoff to systemd, or dbus-broker's unconditional handoff
to systemd.

[1] https://dbus.freedesktop.org/doc/dbus-specification.html

Signed-off-by: Simon McVittie <smcv@debian.org>
---
 virtual-package-names-list.yaml | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/virtual-package-names-list.yaml b/virtual-package-names-list.yaml
index 2a9857a..eb5ace2 100644
--- a/virtual-package-names-list.yaml
+++ b/virtual-package-names-list.yaml
@@ -118,6 +118,10 @@ virtualPackages:
    description: provides the D-Bus well-known session bus for most or all user login sessions
  - name: default-dbus-session-bus
    description: Debian's preferred implementation of dbus-session-bus, possibly architecture-specific
+ - name: dbus-system-bus
+   description: provides the D-Bus well-known system bus as a system service, including service activation
+ - name: default-dbus-system-bus
+   description: Debian's preferred implementation of dbus-system-bus, possibly architecture-specific
  - name: logind
    description: an org.freedesktop.login1 D-Bus API implementation (versioned)
  - name: default-logind
-- 
2.33.1

